src/components/Landing.vue

```diff
@@ -31,40 +40,45 @@
           @keyup.enter="(enteringName = false), (enteringEmail = true)"
         />
         <div
-          id="email-submit"
+          class="email-button"
           @click="(enteringName = false), (enteringEmail = true)"
         >
-          <transition name="fade">
-            <i
-              v-show="name.length >= 2"
-              class="fa fa-arrow-circle-o-right fa-3x"
-              :style="{ alignSelf: 'center' }"
-            ></i>
-          </transition>
+          <i
+            v-show="name.length >= 2"
+            class="fa fa-arrow-circle-o-right fa-3x fade-in"
+            :style="{ alignSelf: 'center' }"
+          />
         </div>
       </div>
-      <transition name="fade">
-        <div v-if="gotit" style="font-family: Montserrat; font-size: 30px;">
-          Got it, thanks!
-        </div>
-      </transition>
       <form v-on:submit.prevent="handleSubmit">
-        <div v-if="enteringEmail && !gotit" class="email-group">
-          <input
-            type="email"
-            placeholder="Your email"
-            id="email-input"
-            v-model="email"
-            required
+      <div v-if="enteringEmail && !gotit" class="email-group">
+        <div
+        class="email-button"
+        @click="(enteringName = true), (enteringEmail = false)"
+        >
+          <i
+              class="fa fa-arrow-circle-o-left fa-3x"
+              :style="{ alignSelf: 'center', marginRight: '19px'}"
           />
-          <button type="submit" id="email-submit">
+        </div>
+        <input
+          type="email"
+          placeholder="Your email"
+          id="email-input"
+          v-model="email"
+          required
+        />
+        <button type="submit" class="email-button">
             <i
               class="fa fa-arrow-circle-o-right fa-4x"
-              :style="{ alignSelf: 'center' }"
-            ></i>
```

Comment:
I think this is missing a `/`

Reply:
The tag is closed, it just looks weird in the change log

Reply:
oh i see, missed line 75 👍